src/stacked_borrows.rs

```diff
@@ -639,7 +646,11 @@ pub trait EvalContextExt<'a, 'mir, 'tcx: 'a+'mir>: crate::MiriEvalContextExt<'a,
     ) -> EvalResult<'tcx> {
         let this = self.eval_context_ref();
         trace!("ptr_dereference: Accessing {} reference for {:?} (pointee {})",
-            if let Some(mutability) = mutability { format!("{:?}", mutability) } else { format!("raw") },
+            if let Some(mutability) = mutability {
+                format!("{:?}", mutability)
+            } else {
+                format!("raw")
+            },
```

Comment:
keep the old code

Reply:
I'm okay with this change but the entire `trace!` call should be block indented since it's using multiple lines.

Reply:
Deal.